installing/installing_azure/installing-restricted-networks-azure-user-provisioned.adoc

```diff
@@ -0,0 +1,147 @@
+:_content-type: ASSEMBLY
+[id="installing-restricted-networks-azure-user-provisioned"]
+= Installing a cluster on Azure in a restricted network with user-provisioned infrastructure
+include::_attributes/common-attributes.adoc[]
+:context: installing-restricted-networks-azure-user-provisioned
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on Microsoft Azure by using infrastructure that you provide.
+
+Several link:https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/overview[Azure Resource Manager] (ARM) templates are provided to assist in completing these steps or to help model your own.
+
+[IMPORTANT]
+====
+The steps for performing a user-provisioned infrastructure installation are provided as an example only. Installing a cluster with infrastructure you provide requires knowledge of the cloud provider and the installation process of {product-title}. Several ARM templates are provided to assist in completing these steps or to help model your own. You are also free to create the required resources through other methods; the templates are just an example.
+====
+
+.Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+* You xref:../../installing/installing_azure/installing-azure-account.adoc#installing-azure-account[configured an Azure account] to host the cluster and determined the tested and validated region to deploy the cluster to.
+* You xref:../../installing/disconnected_install/installing-mirroring-installation-images.adoc#installation-about-mirror-registry_installing-mirroring-installation-images[mirrored the images for a disconnected installation] to your registry and obtained the imageContentSources data for your version of {product-title}.
++
+[IMPORTANT]
+====
+Because the installation media is on the mirror host, you can use that computer to complete all installation steps.
```

Comment:
are they required to use it? the IMPORTANT admonition makes me think they are, in which case I think that should be made clear:
```suggestion
Because the installation media is on the mirror host, you must use that computer to complete all installation steps.
```
But if not, consider using a NOTE admonition instead